flask_rebar/swagger_generation/marshmallow_to_swagger.py

```diff
@@ -604,6 +614,16 @@ def get_maximum(
             return UNSET
 
 
+class ContainsOnlyConverter(ValidatorConverter):
+    MARSHMALLOW_TYPE = ContainsOnly
+
+    @sets_swagger_attr(sw.items)
+    def get_items(
+        self, obj: ContainsOnly, context: _Context
+    ) -> Union[Type[UNSET], m.fields.List]:
+        return {"type": context.memo["items"]["type"], "enum": obj.choices}
```

Comment:
this is messy but I couldn't figure out a better way to figure out the `type` of the parent `value` when the `obj` we are passed is the `marshmallow.validator` i.e. 1 level inside the `value`.